packages/cadl-ranch-specs/http/models/input-basic/mockapi.ts

```diff
@@ -0,0 +1,15 @@
+import { passOnSuccess, ScenarioMockApi, mockapi, ValidationError } from "@azure-tools/cadl-ranch-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+Scenarios.InputBasic_getModel = passOnSuccess(
+  mockapi.get("/input-basic/models", (req) => {
+    if (req.body && req.body["requiredString"] && req.body["requiredInt"]) {
+      return {
+        status: 200,
+      };
+    } else {
+      throw new ValidationError(`Expected InputModel with required properties`, null, null);
```

Comment:
Hi @timotheeguerin  
`ValidationError` (https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-api/src/validation-error.ts) has different format with `ErrorResponse`  defined in `.cadl` file (https://github.com/Azure/cadl-azure/blob/main/packages/cadl-azure-core/lib/foundations.cadl#L71). Is it better to change `ValidationError` format to make it consistent with `ErrorResponse`?